### PR TITLE
Add LFX Mentorship 2023 Summer Projects on :/programs/lfx

### LINKED ISSUE #4208: [Careers] Post New LFX Summer 2023 Internships
### Current Behavior
The [LFX programs page](https://layer5.io/programs/lfx) needs the Summer term listed.

### Desired Situation
Add details of:

- https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md#service-mesh-performance
- https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md#meshery

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by aku1310**
**Description**
- Add 2023 summer projects. 
- Change list type in CNCF Tag Network Spring Project to better differentiate between deliverables and recommended skills, mentors section.

This PR fixes #4208 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b8f827569294ff3b05f3a579e4872c6baaebc70b at: https://645c75e4980a884ecdb4e405--layer5.netlify.app

File: src/collections/programs/lfx-2023/lfx-2023.mdx [hunk 1]
```diff
@@ -113,15 +113,73 @@ A use-case that fits very well is be the ability to use it to run performance te
...
+3. Validation and Reference: For Meshery MeshModel definitions such as cloud-native components and their relationships, the plugin can use the CUE language to provide validation for the CUE input and preview the rendering result. The plugin can also fetch the SMP Model schemas and display them in the IDE for reference.
+
+
+- Recommended Skills: Cuelang
+- Mentor(s): <Link to="/community/members/lee-calcote">Lee Calcote</Link>, <Link to = "/community/members/xin-huang">Xin Huang</Link>
+- Upstream Issue (URL): https://github.com/service-mesh-performance/service-mesh-performance/issues/379
+
+
 ## Additional information
 <br />
```
Comment (vishalvivekm) [inline src/collections/programs/lfx-2023/lfx-2023.mdx:185]: in [Additional information](https://github.com/layer5io/layer5/blob/master/src/collections/programs/lfx-2023/lfx-2023.mdx#additional-information): 
<p>
Our mentors have managed teams of technical writers working on documenting enterprise-grade software at large technology companies (Cisco, Seagate, SolarWinds). During the span of time, <code>he has worked</code> with technical writers in DITA and post-DITA environments (from Word to FrameMaker, structured writing, online help, various CMSes, git). Our mentors have worked with technical writers on documentation strategy, creating document sets, covering the full spectrum of reader personas.
</p>
will you replace the <code>he has worked with </code> to <code>they have worked with</code> @aku1310 ?

**Review by leecalcote — COMMENTED** [review]
Thanks, @aku1310. Will you add an entry for this TAG Network project, too? https://github.com/cncf/mentoring/pull/991/files

**Comment by leecalcote** [comment]
> Thanks, @aku1310. Will you add an entry for this TAG Network project, too? https://github.com/cncf/mentoring/pull/991/files

A second PR can be submitted to tie this off, @aku1310.

**Pull request merged**